src/bowling.js

```diff
@@ -0,0 +1,89 @@
+import Frame from './frame.js';
+
+export default class Bowling {
+  constructor() {
+    this.frames = [];
+    this.gameStatus = true;
```

Comment:
This name "gameStatus", coupled with true is not easy to understand. What is this property indicating? Is it that the game is not finished? If yes, then `this.isGameOver` or `this.hasGameEnded`, and changing your true to false here would be easier to understand.

[Here's an article I wrote about naming](https://medium.com/swlh/an-opinionated-guide-to-naming-your-code-aimed-at-new-developers-6f769d60e098) if you want to go further there.